packages/edit-site/src/components/save-button/index.js

```diff
@@ -53,12 +53,13 @@ export default function SaveButton() {
 	const disabled = ! isDirty || isSaving;
 
 	const [ isOpen, setIsOpen ] = useState( false );
-	const open = useCallback( setIsOpen.bind( null, true ), [] );
-	const close = useCallback( setIsOpen.bind( null, false ), [] );
+	const open = useCallback( () => setIsOpen( true ), [] );
```

Comment:
These were also necessary to edit due to reacts warnings breaking e2e tests.  Without this change, react would throw a warning when the button was used about useState update functions not supporting the second callback argument.